### DISCUSSION TITLE
lowest & highest methods for PricingManager

### DISCUSSION THREAD

**Discussion by poppabear8883** [lunarphp/lunar#190] [Feature Requests]
Often devs will need to know the lowest or highest price from within the collection of variants.

I think we should have methods like `Pricing::lowest($variants)` & `Pricing::highest($variants)`

Example use case would be a product card showing something like `Starting at $x.xx` or `From $x.xx - $xx.xx`

**Comment by zalito12** [comment]
Yeah, it would be great, for now you can add to the formatter or the model by yourself, I did something like this

````php
$prices = $this->variants->pluck('basePrices')->flatten();
$maxPrice = $prices->max('price');
$minPrice = $prices->min('price');
````
